src/js/view/controls/components/menu/menu.js

```diff
@@ -51,9 +51,10 @@ export default class Menu extends Events {
         return quality || captions || audioTracks || sharing || playbackRates;
     }
     createItemsContainer() {
+        const close = this.close;
         const itemsContainerElement = this.el.querySelector('.jw-settings-submenu-items');
         const getTopbar = () => this.topbar;
-        const onKeydown = function(event) {
+        const onKeydown = function(evt) {
```

Comment:
changing this to an arrow func gets rid of any need to cache `this.close` above. given it's the only instance of this, that seems safe

Reply:
Good call